readme.md

```diff
@@ -23,8 +23,13 @@ stringLength('🐴');
 
 stringLength('\u001B[1municorn\u001B[22m');
 //=> 7
+
+stringLength('\u001B[1municorn\u001B[22m', { ignoreEscapeCodes: false });
+//=> 16
 ```
 
+You should only ignore escape codes when working with terminal-related strings; otherwise, you should disable `ignoreEscapeCodes` (which defaults to `true`). This default will change in a future breaking release.
+
```

Comment:
The option needs a API section as done here: https://github.com/sindresorhus/boxen#api